pkg/netflow/flowaggregator/buildpayload.go

```diff
@@ -0,0 +1,76 @@
+package flowaggregator
+
+import (
+	"context"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	coreutil "github.com/DataDog/datadog-agent/pkg/util"
+
+	"github.com/DataDog/datadog-agent/pkg/netflow/common"
+	"github.com/DataDog/datadog-agent/pkg/netflow/payload"
+)
+
+func buildPayload(aggFlow *common.Flow) payload.FlowPayload {
+	var direction string
+
+	if aggFlow.Direction == 0 {
+		direction = "ingress"
+	} else {
+		direction = "egress"
+	}
+
+	hostname, err := coreutil.GetHostname(context.TODO())
+	if err != nil {
+		log.Warnf("Error getting the hostname: %v", err)
+		hostname = ""
+	}
+
+	ipProtocol := fmt.Sprintf("%d", aggFlow.IPProtocol)
+	etherType := fmt.Sprintf("%d", aggFlow.EtherType)
+
+	return payload.FlowPayload{
+		// TODO: Implement Tos
+		FlowType:     string(aggFlow.FlowType),
+		SamplingRate: aggFlow.SamplingRate,
+		Direction:    direction,
+		Exporter: payload.Exporter{
+			IP: aggFlow.ExporterAddr,
+		},
+		Start:      aggFlow.StartTimestamp,
+		End:        aggFlow.EndTimestamp,
+		Bytes:      aggFlow.Bytes,
+		Packets:    aggFlow.Packets,
+		EtherType:  etherType,
+		IPProtocol: ipProtocol,
+		Source: payload.Endpoint{
+			IP:   aggFlow.SrcAddr,
+			Port: aggFlow.SrcPort,
+			// TODO: implement Mac
+			// TODO: implement Mask
+			Mac:  "00:00:00:00:00:00",
```

Comment:
Is this going to be sent to EP?

Reply:
Yes, it's only a fake value for now, it will be fixed in https://github.com/DataDog/datadog-agent/pull/12049